examples/tensorflow-akka/README.md

```diff
@@ -0,0 +1,141 @@
+## `tensorflow-akka`
+
+### Problem Definition
+
+A simple pipeline that scores the quality of wines using a TensorFlow model.
+
+The TensorFlow models must be provided through a persistent volume claim. 
```

Comment:
An interesting alternative would be to use [Alpakka with Google Cloud Storage](https://doc.akka.io/docs/alpakka/current/google-cloud-storage.html)

Reply:
That is an interesting alternative indeed. It might be easier.. I'll have a closer look, thanks @skonto !

The benefit of doing what is done here is that it is a very simple solution combining the volume mount and config features. It also works immediately with runLocal, using local file system. But it does require the PV stuff...

Reply:
I think we could look at this from a separate example.. leaving it for now.